app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -140,4 +150,24 @@ static String readJsonFile (Context context) {
         }
         return resp;
     }
+
+    public static void clickMap(Chat fragment, Activity activity) {
+        RcMaps maps = RcMaps.getInstance();
+        maps.setMapsLocationApiKey(BuildConfig.RC_MAPS_API_KEY)
+                // .setNavigationBarTitleColor(Color.GREEN)
+                // .setNavigationBarBackgroundColor(Color.GREEN)
+                // .setNavigationBarTitleFont(Typeface.DEFAULT_BOLD)
+                // .setNavigationBarIconColor(Color.RED)
+                // .setNavigationBarTitleSize((int) activity.getResources().getDimension(R.dimen.rc_navigation_bar_title_text_size_test))
+                // .setButtonTextSize(12)
+                .build(activity);
+
+        maps.setMapsListener(new RcMaps.RcMapsListener() {
+            @Override
+            public void sendMessageLocation(Intent data) {
+                super.sendMessageLocation(data);
+                Dimelo.getInstance().sendMessageLocation(data, fragment);
+            }
```

Comment:
is this `setMapsListener` part of the code our customer are going to need in order to have a functional map integration?

Reply:
yes